account_renumber_options/wizard/wizard_renumber.py

```diff
@@ -13,31 +13,37 @@ class WizardRenumber(models.TransientModel):
     _inherit = 'wizard.renumber'
 
     @api.multi
+    def get_sequence_for_fiscalyear_id(self, sequence, fiscalyear_id):
+        for line in sequence.fiscal_ids:
```

Comment:
for line in sequence.mapped('fiscal_ids').filtered(lambda x: x.fiscalyear_id.id == fiscalyear_id)